### DISCUSSION TITLE
ERROR: singular Jacobian

### DISCUSSION THREAD

**Discussion by leoforeal** [idaholab/moose#26253] [Q&A General] [answered]
Hallo!

I encounter the following error while running a steady state simulation with solve_type: NEWTON.
**ERROR: Newton scheme encountered a singular Jacobian in element: 156804**

I looked up the meaning of singular Jacobian, which in general means the initial guess causes the solution to diverge. 
Does that mean by adjusting my input parameter in Material or variable could have an effect on the divergence.
Also another thought I have is, if it has something to do with the mesh itself, which in my case contains a fault zone.  

Thanks so much in advance!

**Comment by GiudGiud** [comment]
Hello

Did this after on the first iteration?

Or deep into the solve?

If on the first iteration, this is a problem with the mesh not with the solve.
If not, then the solve is progressing into an unphysical realm (large displacements) that break the mesh

Guillaume

**Comment by leoforeal** [comment]
It happens at the very beginning of the iteration. 
The purpose of the code was to actually carry out refinement process of one of the block in mesh.
In this case, what could be the potential problem of the mesh?

**Reply by GiudGiud** [reply, marked as answer]
Ok so your mesh has issues then.
I would recommend you lead a full diagnostic of your mesh using this mesh generator:
https://mooseframework.inl.gov/source/meshgenerators/MeshDiagnosticsGenerator.html
Then go back to your meshing software to fix the problems.
Unfortunately our in-MOOSE meshing repair is not complete enough (MeshRepairGenerator)
